backend/zeno_backend/routers/chart.py

```diff
@@ -155,7 +155,10 @@ async def update_chart(chart: Chart, project_uuid: str, request: Request):
         request (Request): http request to get user information from.
     """
     await util.project_editor(project_uuid, request)
-    return await update.chart(chart, project_uuid)
+    if chart.project_uuid == project_uuid:
```

Comment:
I *think* you still have to call `select.chart(chart.id)` and then compare that result with the project_uuid. Someone could make a request with a Chart object that has a project_uuid that is different from the charts'

Reply:
The current select.chart takes in two parameters: the project_uuid and the chart_id. Should I implement a select.chart_by_id similar to how select.slice_by_id works? In it, the SQL query could then just take in the chart id and we'd select for the project_uuid.

Reply:
actually you can change `select.chart` to just take the `id`, it doesn't need the UUID. Good catch!

Reply:
we should probably change `slice_by_id` to just be `slice` too

Reply:
Ok let me do that right now!